sql/2020/04_Fonts/04_04.local_vs_host_with_fcp.sql

```diff
@@ -0,0 +1,36 @@
+#standardSQL
+#local_vs_host_with_fcp
+SELECT
+  client,
+  CASE 
+    WHEN NET.HOST(page) != NET.HOST(url) THEN "host"     
+    ELSE "local"
+  END as host_local,
+  COUNT(0) AS freq,
+  SUM(COUNT(0)) OVER (PARTITION BY client) AS total,
+  ROUND(COUNT(0)*100/SUM(COUNT(0)) OVER (PARTITION BY client),2) AS pct,
+  ROUND(COUNTIF(fast_fcp>=0.75)*100/COUNT(0),0) AS pct_fast_fcp,
+  ROUND(COUNTIF(NOT(slow_fcp >=0.25)
+      AND NOT(fast_fcp>=0.75))*100/COUNT(0),0) AS pct_moderate_fcp,
+  ROUND(COUNTIF(slow_fcp>=0.25)*100/COUNT(0),0) AS pct_slow_fcp,      
```

Comment:
Nit: use the [Core Web Vitals](web.dev/vitals) terminology of good, ni (needs improvement), and poor

Also whitespace and `ROUND` suggestions.
```suggestion
  COUNTIF(fast_fcp >= 0.75) / COUNT(0) AS pct_good_fcp,
  COUNTIF(NOT(slow_fcp >= 0.25)
      AND NOT(fast_fcp >= 0.75)) / COUNT(0) AS pct_ni_fcp,
  COUNTIF(slow_fcp >= 0.25) / COUNT(0) AS pct_poor_fcp,      
```

Reply:
Thanks for the clarification.